velox/functions/sparksql/String.h

```diff
@@ -262,6 +262,76 @@ struct EndsWithFunction {
   }
 };
 
+/// substring_index function
+/// substring_index(string, string, int) -> string
+/// substring_index(str, delim, count) - Returns the substring from str before
+/// count occurrences of the delimiter delim. If count is positive, everything
+/// to the left of the final delimiter (counting from the left) is returned. If
+/// count is negative, everything to the right of the final delimiter (counting
+/// from the right) is returned. The function substring_index performs a
+/// case-sensitive match when searching for delim.
+template <typename T>
+struct SubstringIndexFunction {
```

Comment:
Is there a presto sql function in Velox for it, and what's the difference if there is?

Reply:
@rui-mo It seems presto sql does not support this function.

Reply:
Got it, thanks.